prow/gerrit/adapter/adapter.go

```diff
@@ -302,6 +302,8 @@ func (c *Controller) processChange(logger logrus.FieldLogger, instance string, c
 			labels[k] = v
 		}
 		labels[client.GerritRevision] = change.CurrentRevision
+		labels[client.GerritNumber] = string(change.Number)
+		labels[client.GerritPatchset] = string(change.Revisions[change.CurrentRevision].Number)
```

Comment:
this is also somewhat exposed as the last number in the pull refs
but good to expose it separately.

https://github.com/kubernetes/test-infra/blob/c2ff62dee6ae951579467ff221c36598982f600d/prow/apis/prowjobs/v1/types.go#L759-L760